src/main/java/org/testng/xml/Parser.java

```diff
@@ -8,10 +8,7 @@
 import java.io.IOException;
 import java.io.InputStream;
 import java.net.URI;
-import java.util.Collection;
-import java.util.List;
-import java.util.Map;
-import java.util.ServiceLoader;
+import java.util.*;
```

Comment:
The codebase current coding style doesn't use wild cards for imports. Please have this reverted.